packages/tailwindcss-language-server/src/oxide.ts

```diff
@@ -58,10 +58,32 @@ declare namespace OxideV3And4 {
   }
 }
 
+// This covers the Oxide API from v4.1.0+
+declare namespace OxideV5 {
+  interface SourceEntry {
+    base: string
+    pattern: string
+    negated: boolean
+  }
+
+  interface ScannerOptions {
+    sources: Array<SourceEntry>
+  }
+
+  interface ScannerConstructor {
+    new (options: ScannerOptions): Scanner
+  }
+
+  interface Scanner {
+    files: Array<string>
+    sources: Array<SourceEntry>
```

Comment:
Still called `globs`: https://github.com/tailwindlabs/tailwindcss/blob/feat/source-not/crates/node/src/lib.rs#L169-L176